### DISCUSSION TITLE
Installation of Jacktrip without admin privileges

### DISCUSSION THREAD

**Discussion by zantosh** [jacktrip/jacktrip#1119] [Howtos]
I am trying to install this on my laptop where I don't have admin privileges. Is that possible?

**Comment by ntonnaett** [comment]
You can either build it from source and then run it from your build directory. On Linux you could also install the Flatpak from Flathub as a user.

You can find the build instructions in our documentation: https://jacktrip.github.io/jacktrip/Build/Meson_build/

For the Flatpak option, you might be able to just click on Install [here](https://flathub.org/apps/org.jacktrip.JackTrip). If not, add the repo and execute the following from your terminal.
```
flatpak install --user flathub org.jacktrip.JackTrip
```

The Flatpak is meant for graphical usage. The buffer size defaults to 128 and you need to run Pipewire as a sound server (default on recent distributions).

**Comment by mikedickey** [comment]
We also build packages without installers for every release. Go to Releases -> [Release] -> Assets, and:
* on Windows, download the "-binary.zip" file
* on Mac, download the "-signed-application.zip" file

I'm not sure but don't believe these require admin?
